Validator/IsBase64.cs

```diff
@@ -20,27 +20,21 @@ public partial class Validator
         /// <returns></returns>
         public static bool IsBase64(string input)
         {
-            input = SanetizeInput(input);
+            input = SanitizeInput(input);
 
             //Base64 values must be a multiple of 4 characters.
             if (input.Length == 0 || input.Length%4 != 0)
-            {
                 return false;
-            }
 
             //Padding must be 0, 1 or 2 '=' characters.
-            string valueWithoutPadding = input.TrimEnd(paddingCharacter);
+            var valueWithoutPadding = input.TrimEnd(PaddingCharacter);
             if (input.Length - valueWithoutPadding.Length > 2)
-            {
```

Comment:
And I am assuming you are using Resharper? If so, this SO [question](http://stackoverflow.com/q/3622236/1346943) leads to discussion on how to have Resharper automatically add the curly braces back in.